1_date_and_time.py

```diff
@@ -8,20 +8,37 @@
 
 """
 
+from datetime import datetime, timedelta
+
+
+def get_date(count_day=0):
+
+    return datetime.strftime(datetime.now() + count_day * timedelta(days=1), "%d.%m.%Y")
```

Comment:
Функция очень много делает. Она и что-то считает и форматирует. В этом случае оставляем как есть, но обычно можно разнести "форматируем" и "что-то считаем". 

Единственное исправление, можно count_day передавать прям в timedelta. 
```suggestion
    return datetime.strftime(datetime.now() + timedelta(days=count_day), "%d.%m.%Y")
```